src/api/decorators.py

```diff
@@ -1,31 +1,45 @@
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
 
+from cms.models import Region, Language
+
 
 def feedback_handler(func):
     @csrf_exempt
     def handle_feedback(request, region_slug, language_code):
         if request.method != "POST":
             return JsonResponse({"error": "Invalid request."}, status=405)
 
-        data = request.POST.dict()
-        comment = data.pop("comment", "")
-        rating = data.pop("rating", None)
-        category = data.pop("category", None)
+        try:
+            region = Region.objects.get(slug=region_slug)
+            language = Language.objects.get(code=language_code)
```

Comment:
I think it's best practise to only include the parts in the `try`-block which can actually fail with the catched exceptions.
Since the `return JsonResponse` causes following code not to be executed, I see no reason why we would need the whole function inside the `try`-block.